src/popup/index.css

```diff
@@ -25,9 +25,17 @@
     /* Border colors */
     --border-base: 203 213 225;
     --border-focus: 59 130 246;
-    
+
     /* Popup */
     --popup-width: 448px;
     --popup-height: 559px;
   }
 }
+
+input[type='radio']:checked + label span,
+input[type='radio']:checked + span {
+  @apply bg-primary;
+  background-color: bg-primary;
+  box-shadow: 0px 0px 0px 4px white inset;
+  border-color: #3490dc;
+}
```

Comment:
We can move this outside of CSS, depending on how the radio item component will look after changes. We can use the Tailwind's [peer utility](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state) or the [has modifier](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-descendants) since is now available in all browsers.